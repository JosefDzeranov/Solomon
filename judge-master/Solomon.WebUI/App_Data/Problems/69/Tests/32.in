2 98 3249
