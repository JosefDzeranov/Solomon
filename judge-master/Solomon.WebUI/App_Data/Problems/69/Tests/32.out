33.153061224
